compiler/plugins/input/Torch/torch-iree/PluginRegistration.cpp

```diff
@@ -93,16 +93,35 @@ struct TorchSession
     const Dialect *torchConversionDialect = ctx->getLoadedDialect("torch_c");
     const Dialect *tmTensorDialect = ctx->getLoadedDialect("tm_tensor");
 
+    bool hasTorch = false;
+    bool hasOnnx = false;
+
     module.walk([&](Operation *op) {
       Dialect *d = op->getDialect();
       if (d == torchDialect || d == torchConversionDialect) {
-        typeMnemonics.insert("torch");
+        hasTorch = true;
       } else if (d == tmTensorDialect) {
         // TODO: Retire the tm_tensor input pipeline
         typeMnemonics.insert("tm_tensor");
```

Comment:
If we are switching to a bool check we should probably do the same thing for `tmTensor`